src/apps/api/views/competitions.py

```diff
@@ -232,12 +232,14 @@ def collect_leaderboard_data(self, competition):
             leaderID = submission.leaderboard_id
             if leaderID not in leaderboardTitles.keys():
                 leaderboard = Leaderboard.objects.prefetch_related('columns').get(pk=leaderID)
-                columnsTitles.update({leaderID: {col.id: f'{col.title}({col.id})' for col in leaderboard.columns.all()}})
+                columnsTitles.update({leaderID: {}})
                 leaderboardTitles.update({leaderID: f"{leaderboard.title}({leaderID})"})
                 leaderboard_data[leaderboardTitles[leaderID]] = {}
-            leaderboard_data[leaderboardTitles[leaderID]][submission.owner.username] = {}
+            columnsTitles[leaderID].update({f'col:{col.id}-task:{submission.task_id}': f'{col.title}-{col.id}(Task:{submission.task_id})' for col in leaderboard.columns.all()})
```

Comment:
`columnsTitles` -> `columns_titles`
`leaderboardTitles` -> `leaderboard_titles`

snake case in our python is the standard, may have missed a couple other instances, no biggy, we'll try to catch them as we go!